charmhelpers/contrib/openstack/ha/utils.py

```diff
@@ -64,7 +84,83 @@ def update_dns_ha_resource_params(resources, resource_params,
     @param crm_ocf: Corosync Open Cluster Framework resource agent to use for
                     DNS HA
     """
+    _relation_data = {'resources': {}, 'resource_params': {}}
```

Comment:
The doc strings above are now lying to us. Need to sort the doc strings for update_dns_ha_resource_params() and update_hacluster_dns_ha.

Reply:
update_dns_ha_resource_params does what it did before - so I don't think this needs changing?

Reply:
If you expand above, the update_dns_ha_resource_params, starts with "Check for os-*-hostname" but the block of code that corresponds to that is now in update_hacluster_dns_ha.